.phrase.yml

```diff
@@ -0,0 +1,46 @@
+phrase:
```

Comment:
This looks wrong. This file was ignored in the gitignore some time ago so it shouldn't be here. It also doesn't have all the languages and neither the access token (the reason it was ignored) which indicates it's an old, outdated, version. Please remove this file.

Reply:
I don't know why but is has not been in gitignore anymore. Added do list & removed from repo.